internal/security/secretstore/init.go

```diff
@@ -80,6 +83,17 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 	vaultHost := fmt.Sprintf("%s:%v", configuration.SecretService.Server, configuration.SecretService.Port)
 	intervalDuration := time.Duration(b.vaultInterval) * time.Second
 	vc := secretstoreclient.NewSecretStoreClient(lc, req, vaultScheme, vaultHost)
+	pipedHexReader := pipedhexreader.NewPipedHexReader()
+	kdf := kdf.NewKdf(fileOpener, configuration.SecretService.TokenFolderPath, sha256.New)
```

Comment:
is sha256 here good enough? should we be using something like sha3_384 instead ?

Reply:
Yes, SHA256 is sufficient for key deriviation.